sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/LeaseStoreManagerImpl.java

```diff
@@ -370,7 +374,11 @@ public LeaseStoreManagerBuilderDefinition hostName(String hostName) {
                 lease.getId(), new PartitionKey(lease.getId()),
                 this.requestOptionsFactory.createItemRequestOptions(lease),
                 serverLease -> {
-                    if (serverLease.getOwner() != null && !serverLease.getOwner().equalsIgnoreCase(lease.getOwner())) {
```

Comment:
Any test coverage that would need to be added for the scenario?

Reply:
I've added a test case based on the "stale" test, verified that it catches the null Owner case